apps/prairielearn/src/api/v1/endpoints/queries.sql

```diff
@@ -311,6 +311,25 @@ WITH
       s.score,
       s.correct,
       s.feedback,
+      rg.computed_points AS rubric_grading_computed_points,
+      rg.adjust_points AS rubric_grading_adjust_points,
+      (
+        SELECT
+          JSONB_AGG(
+            JSONB_BUILD_OBJECT(
+              'rubric_item_id',
+              rgi.rubric_item_id,
```

Comment:
what is the rubric grading id?

Reply:
There are 4 tables in rubrics:
* `rubrics` correspond to the general rubrics settings for a question;
* `rubric_items` correspond to the rubric items available to be used during grading for a question;
* `rubric_gradings` refers to the grading of one particular submission;
* `rubric_grading_items` refers to the rubric items applied to the grading of one particular submission.

Does this clarify the item? In essence, the `rubric_item_id` should allow you to more easily identify submissions that were graded by applying the same rubric item. This should be safer to use than the description (text), since the text may change with time and there could technically be more than one item with the same text.